files/en-us/web/css/position-visibility/index.md

```diff
@@ -167,24 +167,7 @@ form {
   bottom: 2px;
   right: 2px;
 }
-```
```

Comment:
This property is "more advanced". While it's good to explain all the relevant properties in anchor-name and position-anchor, we don't have to explain every css property on every page. I suggest removing this explanation that is found on the two other pages, but adding the links to those property pages in the see also section.